src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/chatpageobject/ChatPageObject.java

```diff
@@ -243,6 +267,65 @@ public void banUser(String userName) {
     PageObjectLogging.log("clickBanUser", userName + " ban modal is closed", true);
   }
 
+  public void clickOpenUserMessageWall(String userName) {
+    userMessageWallButton.click();
+    PageObjectLogging.log("clickOpenUserMessageWall", "User " + userName + " Message wall button is clicked", true);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "User " 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS1192)